WindowsServerDocs/identity/ad-fs/operations/AD-FS-support-for-alternate-hostname-binding-for-certificate-authentication.md

```diff
@@ -1,38 +1,44 @@
 
-With AD FS in Windows Server 2016 this can be accomplished.
+## AD FS in Windows Server 2016 supports alternate host name binding.
 
-In AD FS on Windows Server 2016 this has changed. Now we support two modes, the first uses the same host (i.e. adfs.contoso.com) with different ports (443, 49443). The second used different hosts (adfs.contoso.com and certauth.adfs.contoso.com) with the same port (443). This will require an SSL certificate to support "certauth.\<adfs-service-name>" as an alternate subject name. This can be done at the time of the farm creation or later via PowerShell.
+AD FS in Windows Server 2016 provides support for alternate host name binding. Two modes are supported. The first mode uses the same host (that is, adfs.contoso.com) with different ports (443, 49443). The second mode uses different hosts (adfs.contoso.com and certauth.adfs.contoso.com) with the same port (443). This will require an SSL certificate to support "certauth.\<adfs-service-name>" as an alternate subject name. This can be done at the time of the farm creation or later via PowerShell.
 
 ## How to configure alternate host name binding for certificate authentication
-There are two ways that you can add the alternate host name binding for certificate authentication. The first is when setting up a new AD FS farm with AD FS for Windows Server 2016, if the certificate contains a subject alternative name (SAN), then it will automatically be setup to use the second method mentioned above. That is, it will automatically setup two different hosts (sts.contoso.com and certauth.sts.contoso.com with the same port. If the certificate does not contain a SAN, then you will see a warning telling you that certificate subject alternative names does not support certauth.*. See the screenshots below. The first one shows an installation where the certificate had a SAN and the second one shows a certificate that did not.
 
-![Screenshot that shows an installation where the certificate had a SAN.](media/AD-FS-support-for-alternate-hostname-binding-for-certificate-authentication/ADFS_CA_1.png)
+There are two ways you can add the alternate host name binding for certificate authentication. 
+
+- When setting up a new AD FS farm with AD FS for Windows Server 2016, if the certificate contains a subject alternative name (SAN), it will automatically be setup to use the second mode mentioned in the preceding section. That is, it will automatically setup two different hosts (sts.contoso.com and certauth.sts.contoso.com) with the same port. 
```

Comment:
```suggestion
- When setting up a new AD FS farm with AD FS for Windows Server 2016, if the certificate contains a subject alternative name (SAN), it will automatically be set up to use the second mode mentioned in the preceding section. That is, it will automatically set up two different hosts (sts.contoso.com and certauth.sts.contoso.com) with the same port. 
```